backend/open_webui/models/chats.py

```
@@ -377,22 +377,47 @@ def archive_all_chats_by_user_id(self, user_id: str) -> bool:
+                    query = query.filter(Chat.title.ilike(f"%{query_key}%"))
+
+                order_by = filter.get("order_by")
+                direction = filter.get("direction")
+
+                if order_by and direction and getattr(Chat, order_by):
+                    if direction.lower() == "asc":
+                        query = query.order_by(getattr(Chat, order_by).asc())
+                    elif direction.lower() == "desc":
+                        query = query.order_by(getattr(Chat, order_by).desc())
+                    else:
+                        raise ValueError("Invalid direction for ordering")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)